lib/solarwinds_apm/patch/tag_sql/annotate_traceparent.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Are these files under the `tag_sql` directory a copy of upstream code?

Reply:
No. Only the query function is prepended with trace context injection logic

Reply:
I see. How about updating the heading comment in these files to use our standard license/copyright wording then--don't want to give the wrong impression.